### ISSUE TITLE
make fails, zlib tar archive not found

### ISSUE THREAD

**Issue by SethosII** [suhrig/arriba#154]
Since the release of `zlib` `1.2.12` a build from source fails because it tries to download `1.2.11` which isn't at the specified location anymore:

```
--2022-04-26 10:46:17--  https://zlib.net/zlib-1.2.11.tar.gz
Resolving zlib.net (zlib.net)... 85.187.148.2
Connecting to zlib.net (zlib.net)|85.187.148.2|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2022-04-26 10:46:18 ERROR 404: Not Found.


gzip: stdin: unexpected end of file
tar: Child returned status 1
tar: Error is not recoverable: exiting now
Makefile:30: recipe for target 'libraries/libz.a' failed
make[1]: *** [libraries/libz.a] Error 2
make[1]: Leaving directory '/dev/shm/arriba_v2.2.1'
Makefile:11: recipe for target 'all' failed
make: *** [all] Error 2
```

**Comment by suhrig** [comment]
Thanks for letting me know. Someone else has noted this recently, too, and submitted a fix. I have not yet found the time to release a new version, but will do so soon. Until then, you can either use the precompiled binary or download the fixed Makefile from here: https://raw.githubusercontent.com/suhrig/arriba/develop/Makefile

I will leave this issue open until the next release in case someone else runs into this issue and seeks for help.

**Comment by suhrig** [comment]
Now fixed in the just released version 2.3.0.
